spring-boot-starter-jgroups/pom.xml

```diff
@@ -59,12 +63,11 @@
         <dependency>
             <groupId>org.jgroups</groupId>
             <artifactId>jgroups</artifactId>
-            <version>3.6.10.Final</version>
+            <version>${jgroups.version}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
-            <version>1.5.3.RELEASE</version>
```

Comment:
Why did you remove the specific version here?

Reply:
because of the nature of the spring boot, all starters version are managed by their parents, so you do not have to specify it explicitly, that's why I removed the specific version.

   <parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>1.5.3.RELEASE</version>
    </parent>

Reply:
Thought that was the case, but wanted to check nonetheless.